SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -51,17 +43,25 @@ class SPATIALGDK_API USchemaDatabase : public UDataAsset
 
 	USchemaDatabase() : NextAvailableComponentId(SpatialConstants::STARTING_GENERATED_COMPONENT_ID) {}
 
-	UPROPERTY(VisibleAnywhere)
-	TMap<FString, FSchemaData> ClassPathToSchema;
+	bool GetComponentIdFromLevelPath(const FString& LevelPath, uint32& OutComponentId) const
```

Comment:
Does this need to be `inline`.

Reply:
Checked - it does not while the method definition is within the class definition.